package.cordovabuild.json

```diff
@@ -8,6 +8,7 @@
     "url": "git+https://github.com/e-mission/e-mission-phone.git"
   },
   "scripts": {
+    "setup-native": "./bin/download_settings_controls.js",
```

Comment:
note to self, this needs to be in `package.json` because it is a javascript hook. It is called from `setup/setup_shared_native.sh`